tests/integration/operator/switch_cr_test.go

```diff
@@ -298,21 +299,34 @@ func cleanupInClusterCRs(t framework.TestContext, cs cluster.Cluster) {
 		t.Logf("failed to list existing CR: %v", err.Error())
 	}
 
-	scopes.Framework.Infof("waiting for pods in istio-system to be deleted")
-	// wait for pods in istio-system to be deleted
+	scopes.Framework.Infof("waiting for workloads in istio-system to be deleted")
+	// wait for workloads in istio-system to be deleted
 	err = retry.UntilSuccess(func() error {
-		podList, err := cs.Kube().CoreV1().Pods(IstioNamespace).List(context.TODO(), metav1.ListOptions{})
```

Comment:
Shouldn't we make use of `metav1.ListOptions{}` to filter by labels? That'd make the changes more simple I think.

Reply:
Thanks, updated.